demo-project/conf/base/catalog_08_reporting.yml

```diff
@@ -37,4 +37,7 @@ reporting.cancellation_policy_grid:
 reporting.confusion_matrix:
   type: matplotlib.MatplotlibWriter
   filepath: ${_base_location}/08_reporting/confusion_matrix.png
+  metadata:
```

Comment:
Is this related to the preview changes?

Reply:
I am not sure of the exact reason why this was added in the PR. But, this seems not related to the preview changes.